File: Inf-121/POO/def5_Lab121/e1_Vehiculo/e1_Vehiculo/Principal.cs
using System;
namespace e1_Vehiculo {
   public class Principal {
      public static void Main() {
         
         Console.WriteLine("\n\t- Datos Estudiante: Cristhian Andres Escobar Herrera");
         Console.WriteLine("\t- CI: 6908109 L.P.");
         
         ArchVehiculo archVh = new ArchVehiculo("archVehiculo.dat");

         //archVh.crear();

         //archVh.adicionar();

         archVh.listar();

         //archVh.Modificar("Toyota");

         //archVh.mostVehiculoPlaca();



         //archVh.contModelo("Corolla");
         

      }
   
   }
}

File: Inf-121/java121-main/src/ejer1/Principal.java
package ejer1;

import java.io.IOException;

public class Principal {

	public static void main(String[] args) throws ClassNotFoundException, IOException {
		
		ArchJuez regJuez = new ArchJuez("archJuez.dat");
		
		regJuez.Crear();
		
		regJuez.Adicionar();
		
		regJuez.Listar();
		
		regJuez.modificar("juan");
		
		regJuez.Eliminar("pepe");
		

	}

}
